### PR TITLE
Outline appears on hover, legend shows

### PR DESCRIPTION
**Pull request by jmithani**
See my comments on the differences below.

### CODE REVIEW

File: leafletjs/data Made/CPS Schools Project/chloropleth.html [hunk 1]
```diff
@@ -113,8 +113,8 @@
 }
 
 var geojson = L.geoJson().addTo(map);
-geojson.addData(geojsonFeature);
-// ... our listeners
+// geojson.addData(geojsonFeature);
+// // ... our listeners
```
Comment (jmithani) [inline leafletjs/data Made/CPS Schools Project/chloropleth.html:118]: `geojsonFeature` was not defined, which threw an error.

File: leafletjs/data Made/CPS Schools Project/chloropleth.html [hunk 2]
```diff
@@ -142,7 +142,7 @@
 var legend = L.control({position: 'bottomright'});
 
 //Legend via Leaflet, place here
-end.onAdd = function (map) {
+legend.onAdd = function (map) {
   var div = L.DomUtil.create('div', 'info legend'),
```
Comment (jmithani) [inline leafletjs/data Made/CPS Schools Project/chloropleth.html:146]: just a typo, you had everything right!

File: leafletjs/data Made/CPS Schools Project/chloropleth.html (hunk 1 above)
Comment (YugHug) [reply leafletjs/data Made/CPS Schools Project/chloropleth.html:118]: i can't tell what you did differently. . . ?

**Review by YugHug — APPROVED** [review]

File: leafletjs/data Made/CPS Schools Project/chloropleth.html [hunk 3]
```diff
@@ -113,8 +113,8 @@
 }
 
 var geojson = L.geoJson().addTo(map);
-geojson.addData(geojsonFeature);
-// ... our listeners
+// geojson.addData(geojsonFeature);
+// // ... our listeners
```
Comment (jmithani) [inline leafletjs/data Made/CPS Schools Project/chloropleth.html:117]: i commented out the line!

**Pull request merged**